packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Client-based rate limits
+excerpt: >-
+  Understand what client-based rate limits at Okta and learn how to design for efficient use of resources
+---
+
+# Client-based rate limits
+
+Client-based rate limiting for the `/authorize` endpoint provides granular isolation between requests made to the `/authorize` endpoint using a combination of the Client ID, user's IP address, and Okta device identifier. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+
+The [`/authorize` endpoint](/docs/reference/api/oidc/#authorize) is the starting point for browser-based OpenID Connect flows such as the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) or the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow). A request to this endpoint authenticates the user and either returns an authorization grant or tokens to the client application as part of the callback response.
+
+Currently, client-based rate limits apply to the OAuth `/authorize` endpoint on both the Okta Org Authorization server and any Custom Authorization Server. All Custom Authorization Servers share a rate limit. The Org Authorization Server has a separate rate limit.
+
+Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket (`/oauth2/{authorizationServerId}/v1` or `/oauth2/v1`). The per minute rate limits on these endpoints apply across an Okta tenant.
+
+For example, company Example.com has 10 OAuth applications running in a production environment. Bob's team is launching a new marketing portal that is a single page OAuth application. Unaware of the rate limits on the `/authorize` endpoint, Bob's team begins running some batch testing scripts against the newly created application that makes hundreds of `/authorize` requests in a single minute. Without the client-based rate limiting framework, the new marketing portal application could potentially consume all of the per minute request limits assigned to Example.okta.com and thereby cause rate-limit violations for the rest of the users that access the other OAuth applications.
```

Comment:
Since the below examples talk about `company.okta.com`, Can we also update it to be `company.com`.

For example, company.com has 10 OAuth applications..